homeassistant/components/upnp/__init__.py

```diff
@@ -162,7 +182,7 @@ def device_discovered(info: Mapping[str, Any]) -> None:
 
 async def async_unload_entry(hass: HomeAssistant, config_entry: ConfigEntry) -> bool:
     """Unload a UPnP/IGD device from a config entry."""
-    _LOGGER.debug("Unloading config entry: %s", config_entry.unique_id)
+    LOGGER.debug("Unloading config entry: %s", config_entry.unique_id)
 
     udn = config_entry.data.get(CONFIG_ENTRY_UDN)
     if udn in hass.data[DOMAIN][DOMAIN_DEVICES]:
```

Comment:
The `async_unload_entry` function needs adjustments

```python
if coordinator := hass.data[DOMAIN].pop(entry.entry_id, None):
    await coordinator.device.async_stop()
```

Reply:
Updated, please check to make sure I've got it right. :-)

Reply:
It certainly looks that way :+1: